include/MNN/Interpreter.hpp

```diff
@@ -249,16 +248,16 @@ class MNN_PUBLIC Interpreter {
      * @param name      given name. if NULL, return first input.
      * @return tensor if found, NULL otherwise.
      */
-    Tensor* getSessionInput(const Session* session, const char* name);
+    Tensor* getSessionInput(const Session* session, const char* name) const;
     /**
      * @brief get output tensor for given name.
      * @param session   given session.
      * @param name      given name. if NULL, return first output.
      * @return tensor if found, NULL otherwise.
      */
-    Tensor* getSessionOutput(const Session* session, const char* name);
+    Tensor* getSessionOutput(const Session* session, const char* name) const;
```

Comment:
We can't modify old api currently. Because some app will only update mnn.so but not compile their code with modified head file.

Reply:
I already reverted all interface changes in "Interpreter.hpp". Since we use MNN in our product, I prefer to commit changes here with PR to avoid unnecessary conflicts with our cloned repository.